utils/oc.py

```diff
@@ -566,3 +584,12 @@ def cleanup(self):
         for oc in self.oc_map.values():
             if oc:
                 oc.cleanup()
+
+
+class OCLogMsg:
```

Comment:
Name is still not in line with the pupose here... don't have a better suggestion though. Maybe `OCFalse`? Not sure :)

Reply:
We are returning a log_level and msg though so why isn't it in line?

Reply:
This class is not a log message, is a class that represents we are not getting an instance of OC. It does carry content used for generating log messages, but that's not the core functionality here. The core functionality is to flag OC was not given.
Anyway, leave it. We can iterate on that, if we ever come to it..

Reply:
I am fine with any name doesn't matter to me, but it made more sense based on the extra info, but you are correct we only return `False`

Reply:
We decided on `OCLogMsg`